app/src/main/res/layout/edit_recipe_bottom_sheet_fragment.xml

```diff
@@ -112,12 +137,35 @@
                     android:id="@+id/purpose"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:inputType="textNoSuggestions"
                     android:enabled="@{vm.recipeBuilder.buildType == BuildType.Purpose}"
+                    android:inputType="textNoSuggestions"
                     android:text="@={vm.recipeBuilder.purpose}" />
 
             </com.google.android.material.textfield.TextInputLayout>
 
+            <com.google.android.material.textfield.TextInputLayout
+                android:id="@+id/sequenceTextInputLayout"
+                isVisible="@{vm.recipeBuilder.buildType != BuildType.Raw}"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="16dp"
+                android:hint="@string/sequence_number"
+                app:helperTextEnabled="true"
+                app:layout_constraintEnd_toEndOf="@id/toggleButton"
+                app:layout_constraintStart_toStartOf="@+id/toggleButton"
+                app:layout_constraintTop_toBottomOf="@+id/toggleButton"
+                app:placeholderText="1"
+                tools:helperText="@string/enter_a_purpose_for_the">
```

Comment:
Isn't this the wrong helper text for a sequence number?

Reply:
Resolved in b9a5c284c641d949af5b6ab5a6e73b8704de6350